portsyncd/portsyncd.cpp

```diff
@@ -250,6 +250,24 @@ void handlePortConfig(ProducerStateTable &p, map<string, KeyOpFieldsValuesTuple>
             if (op == SET_COMMAND)
             {
                 p.set(key, values);
+                for (auto fv : values)
+                {
+                    string field = fvField(fv);
+                    string value = fvValue(fv);
+
+                    /* Update the mtu field on host interface */
+                    if (field == "mtu")
+                    {
+                        char system_cmd[1024];
+                        snprintf(system_cmd, sizeof(system_cmd), "ip link set %s mtu %s", key.c_str(), value.c_str());
+                        int ret_val = system(system_cmd);
```

Comment:
Generally, I've seen the usage of swss::exec to execute commands instead of direct system call. Can you check? Also, you could use stringstream for the command which would be convenient instead of allocating a buffer and then use snprintf

You could refer: IntfMgr::setIntfIp in intfmgr.cpp

Reply:
Thanks Sunny. 

Will check.

Reply:
Hi @prsunny ,

Just updated the diff with exec. Thanks for your suggestion.

Please help review the latest diff.

Cheers,
Haiyang